benchmark/runBenchmarks.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+mvn install && java -jar target/rdf4j-benchmark.jar -wi 10 -i 10 -f 3
+
```

Comment:
It might be better (platform-independent, and easier to do from a CI environment) to just execute the benchmark directly from maven, instead of using a shell script. See http://stackoverflow.com/questions/1089285/maven-run-project for details on how to do that. This would also remove the need to produce a fat jar as you'd just be able to rely on the reactor build path.